python/ctsm/toolchain/gen_mksurfdata_jobscript_multi.py

```diff
@@ -202,25 +198,29 @@ def main():
     # Determine commands for each target list
     # --------------------------
     dataset_dict = {
-        "global-present": (
+        "global-vic": (
             "--start-year 2000 --end-year 2000 --nocrop --vic                  --res",
+            "vic_res",
+        ),
+        "global-potveg": (
+            "--start-year 1850 --end-year 1850 --nocrop --potveg               --res",
+            "f09",
+        ),
+        "global-present": (
+            "--start-year 2000 --end-year 2000 --nocrop                        --res",
             "standard_res_no_crop",
         ),
         "global-present-low-res": (
-            "--start-year 2000 --end-year 2000 --nocrop --vic                  --res",
+            "--start-year 2000 --end-year 2000 --nocrop                        --res",
```

Comment:
Separated `vic_res` from the `standard_res_no_crop` and `low_res_no_crop` resolution groups, because the spreadsheet requests vic for only 3 resolutions.